akka-docs/src/main/paradox/typed/interaction-patterns.md

```diff
@@ -225,6 +225,38 @@ Java
  * There can only be a single response to one `ask` (see @ref:[per session child Actor](#per-session-child-actor))
  * When `ask` times out, the receiving actor does not know and may still process it to completion, or even start processing it after the fact
 
+## Ignoring replies
+
+In some situations you may need to send a message that requires a @scala[`replyTo:ActorRef[T]`]@java[`ActorRef<T> replyTo`], but you are not interested in receiving the reply. In this case, you can pass a @scala[`ActorRef.ignore`]@java[`ActorRef.ignore()`].
+
+@scala[`ActorRef.ignore`]@java[`ActorRef.ignore()`], as the name indicates, returns an `ActorRef` that ignores any message sent to it.
+
+
+With the given protocol and actor behavior:
```

Comment:
What about re-using the request-response sample and referring to that rather than a full protocol just to show of passing ignore? 

"With the same protocol as the @ref[request response](...) above, ignoring the reply would look like this: